components/bunny_grabber.py

```diff
@@ -0,0 +1,32 @@
+import enum
+
+import wpilib
+
+__all__ = ["BunnyGrabber"]
+
+
+class GrabberDirection(enum.Enum):
+    intake = 1
+    release = -1
+    stopped = 0
+
+
+class BunnyGrabber:
+    """Controls the bunny grabber"""
+    bunny_motor = wpilib.Talon
+    direction = GrabberDirection.stopped
+
+    def set_intake(self):
+        self.direction = GrabberDirection.intake
+
+    def set_release(self):
+        self.direction = GrabberDirection.release
+
+    def execute(self):
```

Comment:
Do we care about documenting this function's behavior?

Reply:
All `magicbot` (a RobotPy framework) components implement an `execute` method. I think documenting this particular function is unnecessary because both function and use are dictated by `magicbot`, but we should definitely add an "Intro to Magicbot" page to our wiki.